kafka-admin/.openapi/kafka-admin-rest.yaml

```diff
@@ -756,7 +756,7 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/AclBindingListPage'
-  /rest/acls/resource-operations:
+  /api/v1/acls/resource-operations:
```

Comment:
Is there not a concept like `basepath` in OpenAPI anymore to set it one place the root path?

Reply:
There is, but it's not the convention for other exposed APIs. For example the fleet manager's API:
https://api.openshift.com/?urls.primaryName=kafka%20service%20fleet%20manager%20service

Reply:
ok